### DISCUSSION TITLE
Where can I complete the HIPAA course?

### DISCUSSION THREAD

**Discussion by Kiven668** [MIT-LCP/mimic-code#1329] [MIMIC-General] [answered]
Hi All,
Maybe this is the right place to raise this question. To use the mimic III database, I need to complete the course. 
`the researcher must complete a recognized course in protecting human research participants that includes Health Insurance Portability and Accountability Act (HIPAA) requirements`

But I don't know where can I complete this couse, Could you kindly provide the link or something? Thank you in advance.

Best regards,
Kiven

**Comment by marymlucas** [comment, marked as answer]
Hi, you can follow the instructions here [https://mimic.mit.edu/docs/gettingstarted/](https://mimic.mit.edu/docs/gettingstarted/) it gives you all the steps and links. The required course is offered by CITI (https://about.citiprogram.org/) but you should go step by step from the getting started page so you make sure you select the right course and options.

Hope this is helpful.

Mary

**Reply by Kiven668** [reply]
Thank you, Mary. I have finished the CITI certificate and my account is successfully credentialed. But if I want to access the mimic database, a further course is required, which is typed in my question. There is no link or further information about this course. I found the HIPAA is an American rule from Google and some third parties provide the courses, but I am not sure whether it is the required one for  mimic.

**Reply by marymlucas** [reply]
Hi Kiven
You mentioned Google so I assume you're trying to access the data on BigQuery?  If you're already credentialed shouldn't be any further courses needed to the best of my knowledge.  

Once you sign the data use agreement on the MIMIC-III page on Physionet you should see  the links appear for downloading the data and for requesting access on Google BigQuery.  

See here: https://mimic.mit.edu/docs/gettingstarted/cloud/


Mary

**Reply by tompollard** [reply]
> Once you sign the data use agreement on the MIMIC-III page on Physionet you should see the links appear for downloading the data and for requesting access on Google BigQuery.

Thanks @marymlucas, that's right! @Kiven668 to access datasets on BigQuery, you need to complete the credentialing process, and then click the "Request access on BigQuery" link in the Files section of the relevant project. Currently, only some datasets are available on BigQuery (e.g. MIMIC, eICU Collaborative Research Database).

**Reply by Kiven668** [reply]
Thanks Mary, Thanks Tom. Actually I want to download the data files. 
I have compeleted the credentialing process.
![1656629230175](https://user-images.githubusercontent.com/6001885/176791173-d6e94760-8c1b-4462-a424-6913826d8253.png)
Here are screenshots in my mimicIII page. The required training is HIPAA.
![1656629172233](https://user-images.githubusercontent.com/6001885/176790880-3176c194-d363-41a6-8855-125c25ec6cd4.png)

![1656629211242](https://user-images.githubusercontent.com/6001885/176791184-6b5b8d0e-610f-4a42-92a3-ca82ad35b063.png)

**Reply by tompollard** [reply]
@Kiven668 from the screenshot, it looks like you have not completed the required training (or your training has not yet been approved). You can submit your training report and check the status of the review process at: https://physionet.org/settings/training/.

**Reply by Kiven668** [reply]
you are right, @tompollard. That's my question. I have completed the CITI training. It requires HIPAA training, but I don't know what it is.

**Reply by marymlucas** [reply]
Hi @Kiven668 The CITI training "Data or Specimens Only Research" includes the HIPAA training - there's a module titled "Research and HIPAA Privacy Protections".  I've attached a screenshot of what my CITI gradebook for the required course shows. Besides that I'm also confused and unsure what could be going on, especially since your in screenshot at the top it says your account has been credentialed. It may be you didn't complete all the modules on the CITI training? I hope you're able to get it all figured out.

![Capture](https://user-images.githubusercontent.com/34073785/177597121-efae6918-6d60-409f-b795-214d0a079bd0.PNG)

> you are right, @tompollard. That's my question. I have completed the CITI training. It requires HIPAA training, but I don't know what it is.

**Reply by Kiven668** [reply]
Thank you, @marymlucas. I have completed all the modules, so it is very confusion for me.
![1657147849793](https://user-images.githubusercontent.com/6001885/177656203-476d6d39-3837-45bf-aca6-682c74e5e755.png)

**Reply by tompollard** [reply]
@Kiven668 have you submitted your training for review at: https://physionet.org/settings/training/ ? If so, what is the status of your training (active, expired, etc)? 

I would suggest following the information at https://physionet.org/settings/training/ to request help if necessary. We can check your training status with your username, but it is better to send personal information privately than to post it here.

**Reply by Kiven668** [reply]
Big thanks, Tom. I was not aware of the support request function.
